airgun/views/contentview_new.py

```diff
@@ -271,16 +273,88 @@ def before_fill(self, values=None):
         )
 
 
-class NewContentViewVersionDetailsView(BaseLoggedInView):
+class ContentViewVersionDetailsView(BaseLoggedInView):
     breadcrumb = BreadCrumb()
+    version = Text(locator='.//h2[@data-ouia-component-id="cv-version"]')
+    promoteButton = PF4Button(
+        locator='.//button[@data-ouia-component-id="cv-details-publish-button"]'
+    )
+    editDescription = PF4Button(
+        locator='.//button[@data-ouia-component-id="edit-button-description"]'
+    )
+
+    @View.nested
+    class repositories(Tab):
+        TAB_LOCATOR = ParametrizedLocator(
+            './/button[@data-ouia-component-id="cv-version-details-tabs-tab-repositories"]'
+        )
+        searchbox = PF4Search()
+        table = PatternflyTable(
+            component_id="content-view-version-details-repositories-table",
+            column_widgets={
+                'Name': Text('.//a'),
+                'Version': Text('.//a'),
+                'Release': Text('.//a'),
+                'Arch': Text('.//a'),
+                'Epoch': Text('.//a'),
+            },
+        )
+
+    @View.nested
+    class rpmPackages(Tab):
+        TAB_LOCATOR = ParametrizedLocator(
+            './/button[@data-ouia-component-id="cv-version-details-tabs-tab-rpmPackages"]'
+        )
+        searchbox = PF4Search()
+        table = PatternflyTable(
+            component_id="content-view-version-details-rpm-packages-table",
+            column_widgets={
+                'Name': Text('.//a'),
+                'Type': Text('.//a'),
+                'Product': Text('.//a'),
+                'Content': Text('.//a'),
+            },
+        )
+
+    @View.nested
+    class rpmPackageGroups(Tab):
```

Comment:
can you please remove this camel case  ?

Reply:
> can you please remove this camel case ?

the name of class has grapped from locator value (`TAB_LOCATOR`) 
are we supposed to follow any naming convension @omkarkhatavkar?

Reply:
We dont have the pattern convention that we define to use. It was not a non-blocking comment. it is fine